src/components/VisitorForm.vue

```diff
@@ -92,7 +92,7 @@
                 return this.visitor.address === ''
             },
             invalidPhone() {
-                return this.visitor.phone === '' || !this.visitor.phone.match(/^\d{9,15}$/)
+                return false//this.visitor.phone === '' || !this.visitor.phone.match(/^\d{9,15}$/)
```

Comment:
why you disabled this validation, @hkedia321 ?

Reply:
@pararang had it for testing. Now removed it! You can merge now